phpunit.xml.dist

```diff
@@ -12,26 +17,4 @@
             <directory>./lib/</directory>
         </whitelist>
     </filter>
-
-    <listeners>
-        <listener class="JohnKary\PHPUnit\Listener\SpeedTrapListener" >
-            <arguments>
-                <array>
-                    <element key="slowThreshold">
-                        <integer>200</integer>
-                    </element>
-                    <element key="reportLength">
-                        <integer>10</integer>
-                    </element>
-                </array>
-            </arguments>
-        </listener>
-    </listeners>
-
-    <logging>
-        <log type="coverage-html" target="build/coverage" title="doctrine migrations"
-             charset="UTF-8" yui="true" highlight="true" lowUpperBound="5"
-             highLowerBound="70" />
-        <log type="coverage-clover" target="build/logs/clover.xml"/>
-    </logging>
```

Comment:
@mikeSimonson you can always create your customized `phpunit.xml` locally: it overrides the `.dist` file and allows you to have custom settings.